database/schema/constants/constants.go

```diff
@@ -0,0 +1,10 @@
+package constants
```

Comment:
Wouldn't it be better to add a `constants` directory at the root and create separate files for different types of constants?

Reply:
I think it's ok, the current constants are only used by `schema`.